css/media.css

```diff
@@ -0,0 +1,413 @@
+    margin-left: 13px;
+  }
+  
+  /* ---------------------------------------------- */
+  
+  .section__subtitle {
+    font-size: 30px;
+  }
+
+  .market-section__subtitle {
+    margin: 62px 18px 28px 0;
+  }
+
+
+  .products {
+    justify-content: space-around;
+  }
+
+  .product-card {
+    width: 280px;
+    padding: 0 0 0 34px;
+  }
+
+  .products__product-card:nth-child(2) {
+    padding: 0 0;
+    margin: 0 8px 0 0;
+  }
+
+  .products__product-card:nth-last-child(-n+2) {
+    display: none;
+  }
+  
+  .product-card__picture {
+    margin: 0 0 28px 13px;
+  }
+
+  .product-card__picture img {
+    width: 234px;
+    height: 219px;
+  }
+  
+  .product-card__name {
+    font-size: 20px;
+    margin: 0 0 13px 0;
+  }
+
+  .product-card__price {
+    margin-right: 58px;
+    font-size: 25px;
+  }
+
+  .product-card::after {
+    display: none;
+  }
+
+  .circles {
+    margin-right: 11px;
+  }
+
+
+  .circles__link {
+    width: 20px;
+    height: 20px;
+    margin-left: 6px;
+  }
+
+  .color-option {
+    padding: 0 0 0 0;
+  }
+}
+
+@media screen and (max-width: 620px) {
+
+  .products__product-card:nth-child(2) {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 375px) {
+
+  .container {
+    max-width: 375px;
+  }
+
+  .header {
+    min-height: 53px;
+    padding: 0 28px 0 30px;
+  }
+
+  .header__logo-mobile {
+    margin-top: 20px;
+  }
+
+  .header__logo-mobile img {
+    width: 104px;
+  }
+
+  .burger {
+    width: 41px;
+    height: 5px;
+    margin-top: 9px;
+  }
+
+  .burger::after {
+    margin-top: 8px;
+    width: 41px;
+    height: 6px;
+  }
+
+  
+   /* ------------------------- */
+
+   .market-section {
+    padding-top: 18px;
+    padding-bottom: 100px;
+  }
+  
+  .banner {
+    padding: 50px 0 0 48px;
+    min-height: 192px;
+    background-position: 20% ;
+  }
+  
+  .banner__title {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 19px;
+    max-width: 216px;
+  }
+
+  .banner__link-more {
+    font-size: 12px;
+  }
+  
+  .banner__link-more::after {
+    margin-left: 8px;
+  }
+  
+  /* ---------------------------------------------- */
+  
+  .section__subtitle {
+    font-size: 20px;
+  }
+
+  .market-section__subtitle {
+    margin: 34px 12px 0 0;
+  }
+
+  .product-card {
+    padding: 0 0 0 14px;
+  }
+
+  .products__product-card:nth-last-child(-n+3) {
+    display: none;
+  }
+  
+  .product-card__picture {
+    margin: 0 0 24px 45px;
+  }
+
+  .product-card__picture img {
+    width: 166px;
+    height: 152px;
+  }
+  
+  .product-card__name {
+    font-size: 15px;
+    margin: 0 0 3px 0;
+  }
+
+  .product-card__price {
+    font-size: 20px;
+  }
+
+  .circles {
+    margin-right: 0;
+  }
+
+
+  .circles__link {
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+  }
+
+}
+
+@media screen and (max-width: 320px) {
+
+  .container {
+    max-width: 320px;
+  }
+
+  .header {
+    padding: 0 4px;
+  }
+
+  .header__logo-mobile {
+    margin-top: 19px;
+  }
+/* 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654146)